src/popup/router/components/AuctionCard.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="auction-card">
+    <Avatar :name="name" />
+    <span class="name">{{ name }}</span>
+    <AuctionOverview :name="name" />
+    <ButtonPlain
+      class="help"
+      @click="showHelp"
+    >
+      <QuestionCircle class="icon" />
+    </ButtonPlain>
+  </div>
+</template>
+
+<script>
+import Avatar from './Avatar';
+import ButtonPlain from './ButtonPlain';
+import AuctionOverview from './AuctionOverview';
+import QuestionCircle from '../../../icons/question-circle-border.svg?vue-component';
+import { blocksToRelativeTime } from '../../../filters/toRelativeTime';
+
+export default {
+  components: {
+    Avatar,
+    ButtonPlain,
+    QuestionCircle,
+    AuctionOverview,
+  },
+  props: {
+    name: { type: String, required: true },
+  },
+  methods: {
+    showHelp() {
+      this.$store.dispatch('modals/open', { name: 'auctions-help' });
+    },
+    blocksToRelativeTime,
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@use '../../../styles/variables';
+
+.auction-card {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  align-items: center;
+  width: variables.$extension-width;
+  height: 184px;
+  background-image: url('../../../icons/squares-bg.svg');
+
+  .avatar {
+    width: 48px;
+    height: 48px;
+    box-shadow: rgba(variables.$color-blue, 0.15) 0 0 0 8px;
+    margin-top: 36px;
+  }
+
+  .name {
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 16px;
+    margin-top: 16px;
+  }
+
+  .auction-overview {
+    margin-top: 16px;
+  }
+
+  .help {
+    position: absolute;
```

Comment:
We may just use negative margin to allign `.help`, instead of adding z-index. This won't introduce this level of z-index complexity to an upper elements.